dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java

```diff
@@ -0,0 +1,118 @@
+package com.dotcms.contenttype.business;
+
+import com.dotcms.IntegrationTestBase;
+import com.dotcms.contenttype.model.field.BinaryField;
+import com.dotcms.contenttype.model.field.Field;
+import com.dotcms.contenttype.model.field.FieldVariable;
+import com.dotcms.contenttype.model.field.ImmutableFieldVariable;
+import com.dotcms.contenttype.model.type.BaseContentType;
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.contenttype.model.type.ContentTypeBuilder;
+import com.dotcms.contenttype.model.type.DotAssetContentType;
+import com.dotcms.mock.request.MockAttributeRequest;
+import com.dotcms.mock.request.MockHeaderRequest;
+import com.dotcms.mock.request.MockHttpRequest;
+import com.dotcms.mock.request.MockSessionRequest;
+import com.dotcms.rest.api.v1.temp.DotTempFile;
+import com.dotcms.util.CollectionsUtils;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.portlets.folders.business.FolderAPI;
+import com.dotmarketing.util.FileUtil;
+import com.dotmarketing.util.UUIDGenerator;
+import com.liferay.portal.model.User;
+import com.liferay.portal.util.PortalUtil;
+import com.liferay.portal.util.WebKeys;
+import org.glassfish.jersey.internal.util.Base64;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+
+public class DotAssetBaseTypeToContentTypeStrategyImplTest  extends IntegrationTestBase {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)